### DISCUSSION TITLE
Show changes between simulator versions

### DISCUSSION THREAD

**Discussion by bilalshaikh42** [biosimulations/biosimulations#4360] [Ideas]
A callout box on the view page for a particular simulator/version could show if there are any significant changes such as the addition of algorithms, added/removed parameters etc from the previous version. Tihis would allow for quickly glancing at the view to determine when a particular feature might have been added or removed. 

It would be even nicer to have a full changelog of things such as bugfixes or even commits in the specification. See for example how @renovate-bot displays the changes to a dependency when it bumps them to a new version

**Comment by jonrkarr** [comment]
The specifications could be differenced.

Presently, links to change logs can be included in specifications. Example: https://biosimulators.org/simulators/tellurium. I think this is similar to how PyPI, NPM, etc. manage this.

**Reply by bilalshaikh42** [reply]
Is the tellurium link an example where we already do have change logs included?

**Reply by jonrkarr** [reply]
The tellurium specs include a link to its change log (release notes).

**Reply by bilalshaikh42** [reply]
Oh whoops, I misread that. Yes the link is a good option. It would require a lot of additional coordination I suppose to have a standardized format that would display nicely for each simulator, so maybe the link is the best option for now

**Reply by jonrkarr** [reply]
Differencing specifications is reasonable. I could see highlighting that.

It wouldn't be hard for us to have a field for release notes similar to GitHub. That could be compiled into a log of changes. For tools that already have some form of automated releasing, this doesn't seem hard to support. It could be markdown-formatted just like GitHub so that GitHub users could literally use the same release string.
